lighthouse-core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -23,9 +23,21 @@ writeFileSync(filename, cleanAndFormatLHR(data), 'utf8');
  * @return {string}
  */
 function cleanAndFormatLHR(lhrString) {
+  /** @type {LH.Result} */
   const lhr = JSON.parse(lhrString);
-  delete lhr.configSettings.auditMode;
-  delete lhr.timing;
+
+  // Set to predictable and typical value
+  lhr.configSettings.auditMode = false;
+
+  // Set timing values, which change from run-to-run, to predicable values
```

Comment:
no `-`

Reply:
if we're jumping on the comments there's also a `t` missing from `predictable` :)